.github/README.md

```diff
@@ -44,16 +46,14 @@ $ TFENV_ARCH=amd64 tfenv install 1.3.3
 $ tfenv use 1.3.3
```

Comment:
For which reasons do we have to do the `tfenv` stuff above?

I test with the sample configuration on the `darwin_arm64` architecture and every terraform command worked as expected.

Reply:
I was not able to reproduce the configuration issue I previously had with my new M1 so I guess it was fixed on Terraform's side. I removed the instructions from the README to avoid confusion.